### PR TITLE
feat: use inverted expenses on pnl chart

### PR DESCRIPTION
**Pull request by tomaszantas**
## Description

Show inverted expenses on pnl chart.

## How this has been tested?

image

### CODE REVIEW

File: src/components/ProfitAndLossChart/ProfitAndLossChart.tsx [hunk 1]
```diff
@@ -280,6 +282,19 @@ export const ProfitAndLossChart = () => {
     }
 
     return data
+      ?.map(x => {
+        if (x.operatingExpenses < 0 || x.uncategorizedOutflows < 0) {
```
Comment (doneel) [inline src/components/ProfitAndLossChart/ProfitAndLossChart.tsx:286]: In theory there's a case where `operatingExpenses < 0` but `uncategorizedOutflows > 0` and I think we would want to handle that separately, right?

File: src/components/ProfitAndLossChart/ProfitAndLossChart.tsx [hunk 2]
```diff
@@ -591,6 +606,38 @@ export const ProfitAndLossChart = () => {
           xAxisId='expenses'
           stackId='expenses'
         />
+        <Bar
```
Comment (doneel) [inline src/components/ProfitAndLossChart/ProfitAndLossChart.tsx:609]: After looking at the example image you posted in the PR description (very helpful), I think my preference would be to put the `uncategorizedOutflowsInverse` *above* the categorized expenses and categorized revenue. I think it feels better to have the profit dot visibly sit on top of the `categorized` bars and then have the uncategorized bars higher up.

**Review by doneel — APPROVED** [review]
A couple changes requested, but LGTM!

File: src/components/ProfitAndLossChart/ProfitAndLossChart.tsx (hunk 1 above)
Comment (tomaszantas) [reply src/components/ProfitAndLossChart/ProfitAndLossChart.tsx:286]: Yes, this handles such case. For example: `operatingExpenses < 0` but `uncategorizedOutflows > 0`

image

When both negative: `operatingExpenses < 0` but `uncategorizedOutflows < 0` (after change suggested in the next comment)

image

File: src/components/ProfitAndLossChart/ProfitAndLossChart.tsx (hunk 2 above)
Comment (tomaszantas) [reply src/components/ProfitAndLossChart/ProfitAndLossChart.tsx:609]: I've changed the order, screenshot attached to the previous comment

**Pull request merged**